learn/quick-start.md

```diff
@@ -1,7 +1,142 @@
- - [Running a Local Codex Network with Marketplace Support](https://github.com/codex-storage/nim-codex/blob/master/docs/Marketplace.md)
+### Get Codex binary
+
+ For quick start we would use precompiled binaries from [GitHub release page](https://github.com/codex-storage/nim-codex/releases). If you rather prefer to compile from the sources, please check [Build Codex](/learn/build).
+
+ 1. Download binary and checksum for your platform/architecture
+
+    **Linux/macOS**
+    ```shell
+    version=v0.1.3
+    platform=linux     # darwin
+    architecture=amd64 # arm64
+
+    # Binary
+    curl -LO https://github.com/codex-storage/nim-codex/releases/download/${version}/codex-${version}-${platform}-${architecture}.tar.gz
+
+    # Checksum
+    curl -LO https://github.com/codex-storage/nim-codex/releases/download/${version}/codex-${version}-${platform}-${architecture}.tar.gz.sha256
+    ```
+
+ 2. Verify checksum
+    ```shell
+    # Linux
+    sha256sum -c codex-${version}-${platform}-${architecture}.tar.gz.sha256
+
+    # macOS
+    shasum -a 256 -c codex-${version}-${platform}-${architecture}.tar.gz.sha256
+    ```
+    Make sure you get `OK` in the result
+    ```
+    codex-v0.1.3-linux-amd64.tar.gz: OK
+    ```
+
+ 3. Extract binary
+    ```shell
+    tar -zxvf codex-${version}-${platform}-${architecture}.tar.gz
+    ```
+
+ 4. Copy binary to the appropriate folder
+    ```shell
+    sudo install codex-${version}-${platform}-${architecture} /usr/local/bin/codex
+    ```
+
+ 5. Install dependencies
+    ```shell
+    # Debian-based Linux
+    sudo apt update && sudo apt install libgomp1
+    ```
+
+ 6. Check the result
+    ```shell
+    codex --version
+    ```
+    ```shell
+    Codex version:  v0.1.3
+    Codex revision: 89917d4
+    Nim Compiler Version 1.6.14 [Linux: amd64]
+    ```
+
+ 7. Cleanup
+    ```shell
+    rm -f \
+      codex-${version}-${platform}-${architecture} \
+      codex-${version}-${platform}-${architecture}.tar.gz \
+      codex-${version}-${platform}-${architecture}.tar.gz.sha256
+    ```
+
+
+### Run Codex
+
+ Currently, we can run Codex in two modes
+ - **Client node** - you can upload/download files and purchase storage
+ - **Storage node** - you can sell your local storage
+
+ For quick start we will run Codex client node as an easiest way and if you would like to run a storage node please follow [Run Codex](/learn/run) and [Codex Testnet](/networks/testnet) guides.
+
+ 1. Run Codex client node
+    ```shell
+    codex \
+      --data-dir=./data \
+      --api-cors-origin="*" \
+      --nat=`curl -s https://ip.codex.storage` \
+      --disc-port=8090 \
+      --listen-addrs=/ip4/0.0.0.0/tcp/8070 \
+      --bootstrap-node=spr:CiUIAhIhAiJvIcA_ZwPZ9ugVKDbmqwhJZaig5zKyLiuaicRcCGqLEgIDARo8CicAJQgCEiECIm8hwD9nA9n26BUoNuarCEllqKDnMrIuK5qJxFwIaosQ3d6esAYaCwoJBJ_f8zKRAnU6KkYwRAIgM0MvWNJL296kJ9gWvfatfmVvT-A7O2s8Mxp8l9c8EW0CIC-h-H-jBVSgFjg3Eny2u33qF7BDnWFzo7fGfZ7_qc9P
```

Comment:
These args are good enough for block-exchange. But this will not allow purchasing storage. If we want to enable this, I think we need the eth-provider, eth-key, and marketplace address as well.

Reply:
This is why at the end of this guide we have [Next steps](https://docs-quick-start.codex-docs-8lf.pages.dev/learn/quick-start#next-steps).

<img width="711" alt="Screenshot 2024-09-12 at 10 51 55" src="https://github.com/user-attachments/assets/07140bfb-a06e-4b7d-9c07-d9fad3b54a8b">

<hr>

But I see at least one inconsistency here, in [Run codex](https://5f0080dd.codex-docs-8lf.pages.dev/learn/quick-start#run-codex) paragraph. We should add a note here as well that Marketplace is not covered by that guide.


Do you think we should add all marketplace related things in that "Quick start" guide?

My initial idea was to provide a quick start, so people can `get` Codex, `run` Codex, `play` with Codex.

For advanced use cases like Marketplace, whey would need to follow a long way.